File: diet/src/main/java/com/example/diet/Domain/FamilyInfo.java
package com.example.diet.Domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class FamilyInfo {
    private Long family_id;
    private Long usr_id;
    private String relation;
    private String disease;
    private String allergen;
    private Integer sugar_need;
    private Integer cal_need;
    private Integer fat_need;
}

File: frontend/DayDiet_google/app/src/main/java/com/example/daydiet/model/Domain/FamilyInfo.kt
package com.example.daydiet.model.Domain

import com.example.daydiet.model.entity.BaseResponse


data class FamilyInfo (
    val usr_id : Long,
    val family_id : Int,  //家庭成员id
    val relation : String,     //亲属关系
    val disease : String,   //慢性疾病list
    val allergen : String,   //过敏源list
    val sugar_need : Int,
    val cal_need :Int,
    val fat_need :Int
)

// 获取用户的家庭成员信息  过敏源和疾病
data class FamilyInfoResponse(var data:List<FamilyInfo> ): BaseResponse() {}

